docs/pages/server-access/guides/azure-discovery.mdx

```diff
@@ -0,0 +1,271 @@
+---
+title: Configure Teleport to Automatically Enroll Azure Virtual Machines (Preview)
+description: How to configure Teleport to automatically enroll Azure Virtual Machines.
+---
+
+The Teleport Discovery Service can connect to Azure and automatically
+discover and enroll virtual machines matching configured labels. It will then
+execute an install script on these discovered instances that will install
```

Comment:
```suggestion
execute a script on these discovered instances that will install
```
Since the rest of the sentence indicates what the script will do